packages/medusa/src/services/event-bus.ts

```diff
@@ -99,22 +98,52 @@ export default class EventBusService {
     const cloned = new EventBusService(
       {
         manager: transactionManager,
-        stagedJobRepository: this.stagedJobRepository_,
         jobSchedulerService: this.jobSchedulerService_,
         logger: this.logger_,
         redisClient: this.redisClient_,
         redisSubscriber: this.redisSubscriber_,
+        cacheService: this.cacheService_,
       },
       this.config_,
       false
     )
 
-    cloned.transactionManager_ = transactionManager
     cloned.queue_ = this.queue_
 
     return cloned
   }
 
+  /**
+   * Processes jobs in the queue.
+   * @param job The job object
+   * @return resolves to the results of the subscriber calls.
+   */
+  worker_ = async <T>(job: {
```

Comment:
comment: Note this is left unchanged. Changed the structure of the service, which is why is looks like new changes